website/content/docs/get-started/developer-qs.mdx

```diff
@@ -321,6 +321,20 @@ System.out.println("Secret written successfully.");
 
 </CodeBlockConfig>
 
+<CodeBlockConfig lineNumbers>
+
+```Curl
```

Comment:
This language keyword determines the syntax highlighting, but looking at the Vercel deployment I don't think the highlighting is working. 

I'm not actually sure where the list is for valid language keywords to put here even is located, but it looks like you'll need to find the correct keyword. (For Go and Ruby we seem to be using a lower-case one... so maybe it's curl with a lower-case c?) And maybe there is no curl-specific one and you'll need to use a bash or sh one or something?

Reply:
aha. I thought maybe that was for the tab name, but this makes sense. I can dig for a list of language keywords.

Reply:
I know it's something specific to the "flavor" of markdown we're using, rather than something stored in our codebase. I'm just not sure which flavor that is, or where the corresponding list would be. I bet Loann or Yoko would know!

Reply:
Got it! Thanks!